src/storage/read_pool.rs

```diff
@@ -1,5 +1,7 @@
 // Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
 
+//! Distinct thread pools to handle read commands having different priority levels
```

Comment:
This is already described by the latter doc.

Reply:
Same as with the doc comment for errors.rs. I added this after looking at other top level comments present in TiKV/storage - https://tikv.github.io/doc/tikv/storage/index.html (config, types). I can remove it if you feel it's not needed.